pkgs/tools/filesystems/glusterfs/default.nix

```diff
@@ -69,17 +69,6 @@ rec {
     sed -e '/chmod u+s/d' -i contrib/fuse-util/Makefile.am
   '';
 
-  patches = [
-    # Remove when https://bugzilla.redhat.com/show_bug.cgi?id=1450546 is fixed
-    ./glusterfs-use-PATH-instead-of-hardcodes.patch
-    # Remove when https://bugzilla.redhat.com/show_bug.cgi?id=1450593 is fixed
-    ./glusterfs-python-remove-find_library.patch
-    # Remove when https://bugzilla.redhat.com/show_bug.cgi?id=1489610 is fixed
-    ./glusterfs-fix-bug-1489610-glusterfind-var-data-under-prefix.patch
```

Comment:
The issue https://bugzilla.redhat.com/show_bug.cgi?id=1489610 is still open.

Was this really fixed?

Reply:
Just make sure I understand the bug correctly: When you run glusterd (and run actions such as creating volumes) it would crash because it would try to write to the nix store instead of /var.

If that is the case, then yes, the bug is fixed now.

Reply:
> When you run glusterd (and run actions such as creating volumes)

No, that's not what the bug says. It specifically mentions `glusterfind` and also shows the concrete invocation of that tool that results in the failure.

Reply:
OK, that problem has indeed not been fixed. I had to adapt the patch, since now things would end up in `/var/var/lib...`  and remove `$out/bin/conf.py` since that would be wrongfully loaded by `glusterfind` and make it fail.